packages/location-management/src/ducks/location-units.ts

```diff
@@ -41,7 +41,7 @@ export interface LocationUnit {
   properties: Properties;
   syncStatus: string;
   type: string;
-  locationTags: LocationUnitTag[];
```

Comment:
we are receiving locationTags inside payload how will we deal with that over here ?

Reply:
In that case we don't have any actions in this and all the locations, we will handle that while sending data to backend or receiving data, we change the key for ui

Reply:
yes you dont need to modify these keys